modules/user_accounts/jsx/permissions.js

```diff
@@ -0,0 +1,337 @@
+/* global ReactDOM */
+
+var Role = React.createClass({
```

Comment:
Ya, I actually wrote this almost 6 months ago for CCNA so we weren't using that at the time...

Probably not gonna update this now, but thanks for all your comments! Super helpful for a React noob like me :)